tests/particle_interpolator_quadratic_least_squares_2d_limiter.prm

```diff
@@ -0,0 +1,122 @@
+# A test based on the particle version of the SolKz benchmark that (1) tests to
```

Comment:
Update this text. Also it looks like this is nearly identical to the other test above. If so, you can use `include $ASPECt-SOURCE_DIR/tests/particle_interpolator_quadratic_least_squares_2d_add_particles.prm` to avoid most of the lines in the .prm file (except for Dimension, and Additional shared libraries).